projects/angular/src/layout/vertical-nav/_properties.vertical-nav.scss

```
@@ -49,13 +49,14 @@
     --clr-vertical-nav-toggle-icon-hover-color: #{tokens.$cds-alias-object-interaction-color-hover};
     --clr-vertical-nav-toggle-icon-active-color: #{tokens.$cds-alias-object-interaction-color-active};
     --clr-vertical-nav-toggle-icon-disabled-color: #{tokens.$cds-alias-object-interaction-color-disabled};
-    --clr-vertical-nav-toggle-button-size: #{tokens.$cds-global-space-10};
+    --clr-vertical-nav-toggle-button-size: #{tokens.$cds-global-space-11};
     --clr-vertical-nav-trigger-divider-border-color: #{tokens.$cds-alias-object-border-color}; // var(--clr-color-neutral-400);
     --clr-vertical-nav-trigger-divider-border-width: var(--clr-global-borderwidth);
     --clr-vertical-nav-trigger-collapsed-margin: mixins.baselinePx(3);
 
     // Vertical Nav - Header
     --clr-vertical-nav-header-font-weight: #{tokens.$cds-global-typography-font-weight-semibold};
     --clr-vertical-nav-header-padding: #{tokens.$cds-global-space-5} #{tokens.$cds-global-space-7};
+    --clr-vertical-nav-header-line-height: #{tokens.$cds-global-space-7};
```

Set `clr-vertical-nav-header-padding` top and bottom to 10px and remove the line height token.


```suggestion
    --clr-vertical-nav-header-padding: #{mixins.baselinePx(10)} #{tokens.$cds-global-space-7};
```